views/cup_groups/positions.html.erb

```diff
@@ -1,111 +1,23 @@
-<h2><%= I18n.t('.positions.title') %></h2>
-<div class="row">
-  <div class="col-md-8">
-    <% groups.each do |group| %>
-      <div class="panel panel-success">
-        <div id="group-<%= group.name.downcase %>" class="panel-heading">
-          <h3 class="panel-title"><%= "#{I18n.t('.common.group')} #{group.name}" %></h3>
-        </div>
-        <div class="panel-body">
-          <table class="table table-striped table-hover position-table">
-            <thead>
-              <tr>
-                <th><%= I18n.t('.positions.headers.pos') %></th>
-                <th><%= I18n.t('.positions.headers.team') %></th>
-                <th><%= I18n.t('.positions.headers.points') %></th>
-                <th><%= I18n.t('.positions.headers.played') %></th>
-                <th><%= I18n.t('.positions.headers.won') %></th>
-                <th><%= I18n.t('.positions.headers.draw') %></th>
-                <th><%= I18n.t('.positions.headers.lost') %></th>
-                <th><%= I18n.t('.positions.headers.goals') %></th>
-                <th><%= I18n.t('.positions.headers.goals_against') %></th>
-                <th><%= I18n.t('.positions.headers.goal_difference') %></th>
-              </tr>
-            </thead>
-            <tbody>
-              <% group.positions_table.each_with_index do |position, index| %>
-                <tr>
-                  <td class="position-column">
-                    <span class="label label-<%= class_for_index(index) %>">#<%= index + 1 %></span>
-                  </td>
-                  <td class="team-column">
-                    <span><img class="flag" src="/img/flags/<%= position.team.flag %>"></span>
-                    <span class="team-name"><a href="/teams/<%= position.team.iso_code %>"><%= position.team.name %></a></span>
-                  </td>
-                  <td class="points-column">
-                    <%= position.points %>
-                  </td>
-                  <td class="played-column">
-                    <%= position.won + position.lost + position.tied %>
-                  </td>
-                  <td class="won-column">
-                    <%= position.won %>
-                  </td>
-                  <td class="draw-column">
-                    <%= position.tied %>
-                  </td>
-                  <td class="lost-column">
-                    <%= position.lost %>
-                  </td>
-                  <td class="goals-column">
-                    <%= position.goals %>
-                  </td>
-                  <td class="goals-against-column">
-                    <%= position.received_goals %>
-                  </td>
-                  <td class="goal-difference-column">
-                    <%= position.values[:diff] %>
-                  </td>
-                </tr>
-              <% end %>
-            </tbody>
-          </table>
+<%
+content_for :page_title do
+  I18n.t('.positions.title')
+end
+%>
```

Comment:
@kandalf I know you already updated the header to yield page title, but I'm not sure what's the key for it.   Could you provide an example  for this?

Reply:
@tarolandia this is exactly what I did on the template I'm working on:
```Ruby
<% content_for(:title){ I18n.t('.common.fixture') } %>

<div class="row">
  <% groups.each do |group_name, matches| %>
    <div class="col-xl-6 col-lg-12">
```

If you replace the key name, and leave it as is, it will work as soon as we merge my work, which should be done between today and tomorrow, tops.